blueprints/data-solutions/gcs-to-bq-with-least-privileges/main.tf

```diff
@@ -120,19 +79,14 @@ module "project" {
     "storage.googleapis.com",
     "storage-component.googleapis.com",
   ]
-
-  # additive IAM bindings avoid disrupting bindings in existing project
-  iam          = var.project_create != null ? local.iam : {}
-  iam_additive = var.project_create == null ? local.iam : {}
-  shared_vpc_service_config = local.shared_vpc_project == null ? null : {
+  group_iam    = local.group_iam
```

Comment:
`group_iam` is authoritative, so distinction below between `iam` and `iam_additive` will not suffice

Reply:
moved all to iam and iam_additive